yt/geometry/grid_geometry_handler.py

```diff
@@ -128,7 +128,7 @@ def _initialize_level_stats(self):
         #   1 = number of cells
         #   2 = blank
         desc = {'names': ['numgrids','numcells','level'],
-                'formats':['Int64']*3}
+                'formats':['int64']*3}
```

Comment:
As a note, this is some of the very earliest code written in yt.  It probably could be improved in 4.0.